port/sockets.c

```diff
@@ -1024,12 +1030,13 @@ static int do_getifaddrs(char *buf, size_t *buflen)
 	int i;
 #endif
 
-	NETIF_FOREACH(netif) {
+	NETIF_FOREACH(netif)
+	{
```

Comment:
Hmm, why this change? It adhered to our style

Reply:
Apparently this is how our clang-format does it (I just ran it on this file), but I'll change it manually

Reply:
foreach-alike macros need to be explicitly configured in clang-format (https://clang.llvm.org/docs/ClangFormatStyleOptions.html#foreachmacros) - then it would be formatted correctly by default :)